builders/build/eth-api/dev-env/tenderly.md

```diff
@@ -0,0 +1,103 @@
+2. Select **Moonbeam** or **Moonriver** from the **Network** menu, adjust any of the compilation settings, and specify the block to run your code on if needed
+3. Update the JavaScript editor on the right-hand side for your contract. Ethers.js and Web3.js are included in the Sandbox by default and can be instantiated with `ethers` and `web3`, respectively. It's also important to note that the Sandbox includes [global variables](https://docs.tenderly.co/tenderly-sandbox#available-javascript-global-variables){target=_blank} to ease development, so you don't need to worry about updating the RPC URL for Moonbeam
+4. Click on **RUN** when you're ready to compile your contract and execute your code
+
+If your code contained logic to deploy your contract or send a transaction, you'll see the transaction(s) appear under the **Simulated Transactions** section on the bottom left-hand side. 
+
+![Tenderly Sandbox](/images/builders/build/eth-api/dev-env/tenderly/tenderly-2.png)
+
+### Add a Contract {: #add-a-contract }
+
+A good place to start with the Tenderly dashboard is to add a deployed smart contract. Once you've added a contract, you'll be able to create transaction simulations and forks, use the debugger, set up monitoring and alerts, and more. 
+
+To add a new contract, you can click on **Contracts** on the left-side panel and click **Add Contract**. A pop-up will appear and you can take the following steps:
+
+1. Enter the contract address
+2. Choose **Moonbeam** or **Moonriver** as the network, depending on which network you've deployed your smart contract to
+3. (Optional) You can give your contract a name
+4. (Optional) You can toggle the **Add more** slider to on if you'd like to add additional contracts. This will allow you to add more contracts after the initial contract has been added
+5. Finally to add the contract to the dashboard, click **Add contract**
+
+![Add a contract](/images/builders/build/eth-api/dev-env/tenderly/tenderly-3.png)
+
+After a contract has been added, it will appear in the list of contracts on the **Contracts** dashboard. If the contract hasn't been verified yet, the dashboard will display an **Unverified** status along with a **Verify** button.
+
+![Contract in list of contracts](/images/builders/build/eth-api/dev-env/tenderly/tenderly-4.png)
+
+To take full advantage of the Tenderly tool set, it is recommended that you verify your smart contracts, which you can do by clicking on **Verify**. You can choose to verify your contract by uploading the contract's JSON, ABI, or source code. For more information, please refer to [Tenderly's documentation](https://docs.tenderly.co/monitoring/smart-contract-verification/verifying-a-smart-contract#verifying-a-smart-contract){target=_blank}.
+
+### Create a Fork {: #fork-moonbeam }
+
+Tenderly's forking feature simulates the live Moonbeam network in an isolated environment, which enables you to interact with deployed contracts and live on-chain data.
```

Comment:
Same as above, I would explain that Tenderly forks only support the Ethereum Compatibility layer of Moonbeam and not precompiles.